pepper-apis/src/test/java/org/broadinstitute/ddp/route/ProfileRouteTest.java

```diff
@@ -35,6 +37,8 @@
     private static final Integer birthMonth = 3;
     private static final Integer birthDayInMonth = 15;
     private static final Integer birthYear = 1995;
+    private static final LocalDate birthDate = LocalDate.of(1995, Month.MARCH, 15);
+    private static final String invalidBirthDate = "1999-29-09"; //yyyy-mm-dd
```

Comment:
Is this format string the expected format of the date, or the format of the birth date string?

Reply:
I just mentioned it there as format of the date; 29 invalid month